CHANGES.md

```diff
@@ -9,7 +9,8 @@
   * `ThresholdOptmizer` is now a scikit-learn meta-estimator, and accepts
     an estimator through the `estimator` parameter. To use a pre-fitted
     estimator, pass `prefit=True`.
-* Rename arguments of `create_group_metric_set()` to match the dashboard
+* Add `create_dashboard_dictionary()` for creating the Dashboard cache layer
+* Make `create_group_metric_set()` private and change arguments
```

Comment:
Does that break anyone who's been using it before?

Reply:
Yes, although I somehow doubt that the userbase is larger than "me."

Reply:
Fair. If we make such changes in the future we should ideally keep it around for a release longer and issue a warning message indicating the deprecation and impending removal. Maybe that's a little over the top for this one, but just throwing this out there.

Reply:
Agree in general - hence our elaborate migration guide from 0.2.

In this specific case, this functionality is only useful for generating a JSON of the Dashboard cache layer, and right now, you can't even load it back into the dashboard. `_create_group_metric_set()` is only of use for AzureML, and we're still in exploratory stages for that.